### ISSUE TITLE
How to add momentum and weight decay function to config file

### ISSUE THREAD

**Issue by sumitra20** [experiencor/keras-yolo3#228]
Hi,

I would like to know more about the momentum and weight decay function. Is it  possible to add two additional parameters as mentioned below in the config file:

momentum = 0.9
weight decay= 5×10−4

Response will be much appreciated. Thank you
